### PR TITLE
Adds snippets for tables/charts/graphs in Predictions section of en.js

### PR DESCRIPTION
**Pull request by ValerieKenyon**
This adds my recent snippets for Predictions from the en.js Google Doc.  Sorry that I did this in reverse workflow.

### CODE REVIEW

File: src/localization/en.js [hunk 1]
```diff
@@ -269,6 +269,41 @@ export default {
       chartsHeader_v1:            `With the new pay, how could your benefits change?`,
       warningMessage_v1:          `This tool is in testing and these numbers might not be right. If they're not, we'd appreciate your feedback.`,
       submitFeedback_v1:          `Submit Feedback`,
+      // added the following snippets Friday 2018_9_7
```
Comment (knod) [inline src/localization/en.js]: Heh, I appreciate the attention to detail, but this comment isn't neccessary.

File: src/localization/en.js [hunk 2]
```diff
@@ -269,6 +269,41 @@ export default {
...
+      columnCurrentBenefits_v1:   `Current Benefits`,
+      columnNewEstimate_v1:       `New Estimate`,
+      columnDifference_v1:        `Difference`,
+      rowSNAP_v1:                 `SNAP`,
+      rowSection8_v1:             `Section 8 Housing`,
+      rowTotalBenefits_v1:        `Total Benefits`,
+      rowIncome_v1:               `Income`,
+      rowNetTotal_v1:             `Net Total`,
+      perMonth_v1:                ` / month`,
+      dollarSign_v1:              `$`,
```
Comment (knod) [inline src/localization/en.js]: Am I off-base, or does this use the changes introduced here: https://github.com/codeforboston/cliff-effects/pull/730/files#diff-2961681bf8b9ccd9382824bc4430f707R268?

**Comment by knod** [comment]
Also, you might want to see how this is breaking the 'Predictions' snapshot. Doing `npm run test` may give you more info on what's going on.

File: src/localization/en.js (hunk 2 above)
Comment (pporfilio) [reply src/localization/en.js]: My PR did introduce a subset of these changes (I copied the snippets I needed from the google doc). My vote is on rebasing whichever ends up getting merged second. Feel free to ping me when you get to that point and I'd be happy to do it.

**Comment by ValerieKenyon** [comment]
Thanks for your help Parker!

**Pull request merged**